browser/scss/crib/main.scss

```diff
@@ -33,6 +22,50 @@
     }
 }
 
+.normal-background {
+    background: url('http://i.imgur.com/d8C45QS.png') no-repeat center fixed;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+    position: fixed;
+    top: 0;
+    left: 0;
+    min-width: 100%;
+    min-height: 100%;
+    z-index: -1;
+}
+
+.space-background {
+    background: url("http://i.imgur.com/cqOaGQe.png") no-repeat center fixed;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+    position: fixed;
+    top: 0;
+    left: 0;
+    min-width: 100%;
+    min-height: 100%;
+    z-index: -1;
+}
+
+.brick-background {
+    background: url("http://i.imgur.com/Hv9Be1e.png") no-repeat center fixed;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+    position: fixed;
+    top: 0;
+    left: 0;
+    min-width: 100%;
+    min-height: 100%;
+    z-index: -1;
+}
+
+
+
 .dock {
```

Comment:
nice. this looks so much better. maybe have a plain "background-change" class that has the same repetitive styling. and have a unique background "brick" or "space" id that has the unique background url